Lib/http/server.py

```diff
@@ -1336,10 +1355,7 @@ def test(HandlerClass=BaseHTTPRequestHandler,
             print("\nKeyboard interrupt received, exiting.")
             sys.exit(0)
 
-if __name__ == '__main__':
-    import argparse
-    import contextlib
-
+def _main(args=None):
```

Comment:
you are right that you separated this into a separate function `_main` but the imports should be returned, they were specially placed here
```suggestion
def _main(args=None):
    import argparse
    import contextlib

```